typings/omni-fetch.d.ts

```diff
@@ -0,0 +1,3 @@
+declare module 'omni-fetch' {
+  export default fetch;
```

Comment:
That boils down to moving the contents of `isomorphic-fetch`, right? do you want to keep the `isomorphic-fetch` typings as they are or should I delete them?